src/legacy/core_plugins/interpreter/common/index.ts

```diff
@@ -17,19 +17,20 @@
  * under the License.
  */
 
-export const error = () => ({
-  name: 'error',
-  to: {
-    render: input => {
-      const { error, info } = input;
-      return {
-        type: 'render',
-        as: 'error',
-        value: {
-          error,
-          info,
-        },
-      };
-    },
-  },
-});
+export {
+  Datatable,
+  DatatableColumn,
+  DatatableRow,
+  DatatableColumnType,
+  Filter,
+  InterpreterError,
+  isDatatable,
+  KibanaContext,
+  KibanaDatatable,
+  PointSeries,
+  PointSeriesColumns,
+  PointSeriesColumn,
+  PointSeriesColumnName,
+  Render,
+  Style,
+} from './types';
```

Comment:
umm, I have no idea why this diff is showing the error file being moved. these are two separate things.